### PR TITLE
Updates including stakeholders feedback

### PR DESCRIPTION
**Pull request by abhinavs95**
Following changes are made:
* Added ONNX version for each model
* Removed MMS archives, code snippets and descriptions
* Removed MXNet code snippets
* Added license for each model class
* Changed top-level classifications to model application
* Added sections for use cases for each model class

### CODE REVIEW

File: models/image_classification/mobilenet/README.md [hunk 1]
```diff
@@ -23,79 +28,20 @@ The inference was done using jpeg image.
 ### Preprocessing
 The images have to be loaded in to a range of [0, 1] and then normalized using mean = [0.485, 0.456, 0.406] and std = [0.229, 0.224, 0.225]. The transformation should preferrably happen at preprocessing.
```
Comment (ankkhedia) [inline models/image_classification/mobilenet/README.md]: Add link to show preprocessing, should be a different file

File: models/image_classification/mobilenet/README.md [hunk 2]
```diff
@@ -23,79 +28,20 @@ The inference was done using jpeg image.
...
-    return img
-    
- ```
- 
-
 ### Output
 The model outputs image scores for each of the [1000 classes of ImageNet](../synset.txt).
 
 ### Postprocessing
 The post-processing involves calculating the softmax probablility scores for each classes and sorting them to report the most probable classes.
```
Comment (ankkhedia) [inline models/image_classification/mobilenet/README.md]: Add link to file

**Comment by abhinavs95** [comment]
@ankkhedia pushed changes including your feedback

File: README.md [hunk 3]
```diff
@@ -7,7 +7,7 @@
 	<img src="images/ONNX Model Zoo Graphics.png" width="60%"/>
 </p>
 
-The ONNX Model Zoo is a collection of pre-trained models for state of the art models in deep learning, available in the ONNX format. Where supported, the models are also available in the  [Model Server](https://github.com/awslabs/mxnet-model-server) archive format. Accompanying each model are [Jupyter](http://jupyter.org) notebooks for model training and running inference with the trained model. The notebooks are written in Python and include links to the training dataset as well as references to the original paper that describes the model architecture. The notebooks can be exported and run as python(.py) files.
+The ONNX Model Zoo is a collection of pre-trained models for state of the art models in deep learning, available in the ONNX format. Accompanying each model are [Jupyter](http://jupyter.org) notebooks for model training and running inference with the trained model. The notebooks are written in Python and include links to the training dataset as well as references to the original paper that describes the model architecture. The notebooks can be exported and run as python(.py) files.
```
Comment (simoncorstonoliver) [inline README.md]: state-of-the-art

File: models/image_classification/mobilenet/README.md [hunk 4]
```diff
@@ -1,14 +1,19 @@
...
+
+## Use cases
+Mobilenet models perform image classification - they take images as input and classifies the major object in the image into a set of pre-defined classes. They are trained ImageNet dataset which contains images from 1000 classes. MobileNet models are also very efficient in terms of speed and size and hence are ideal for embedded and mobile applications.
```
Comment (simoncorstonoliver) [inline models/image_classification/mobilenet/README.md]: classifies --> classify

are trained Imagenet --> are trained on Imagenet

File: models/image_classification/mobilenet/README.md [hunk 5]
```diff
@@ -1,14 +1,19 @@
...
+
+## Description
+MobileNet improves the state of the art performance of mobile models on multiple tasks and benchmarks as well as across a spectrum of different model sizes. MobileNet is based on an inverted residual structure where the shortcut connections are between the thin bottleneck layers. The intermediate expansion layer uses lightweight depthwise convolutions to filter features as a source of non-linearity. Additionally,  it removes non-linearities in the narrow layers in order to maintain representational power.
```
Comment (simoncorstonoliver) [inline models/image_classification/mobilenet/README.md]: state-of-the-art

File: models/image_classification/mobilenet/README.md [hunk 6]
```diff
@@ -21,81 +26,22 @@ All pre-trained models expect input images normalized in the same way, i.e. mini
...
-    transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])
-    ])
-    img = transform_fn(img)
-    img = img.expand_dims(axis=0) # batchify
-    
-    return img
-    
- ```
- 
+The images have to be loaded in to a range of [0, 1] and then normalized using mean = [0.485, 0.456, 0.406] and std = [0.229, 0.224, 0.225]. The transformation should preferrably happen at preprocessing. Check [imagenet_preprocess.py](../imagenet_preprocess.py) for code.
```
Comment (simoncorstonoliver) [inline models/image_classification/mobilenet/README.md]: do you mean [0, 1) ?

File: models/image_classification/resnet/README.md [hunk 7]
```diff
@@ -1,5 +1,10 @@
...
+
+## Use cases
+ResNet models perform image classification - they take images as input and classifies the major object in the image into a set of pre-defined classes. They are trained ImageNet dataset which contains images from 1000 classes. ResNet models provide very high accuracies with affordable model sizes. They are ideal for cases when high accuracy of classification is required.
```
Comment (simoncorstonoliver) [inline models/image_classification/resnet/README.md]: classifies --> classify

trained ImageNet --> trained on the ImageNet

File: models/image_classification/resnet/README.md [hunk 8]
```diff
@@ -1,5 +1,10 @@
...
+
+## Description
+Deeper neural networks are more difficult to train. Residual learning framework ease the training of networks that are substantially deeper. The research explicitly reformulate the layers as learning residual functions with reference to the layer inputs, instead of learning unreferenced functions. It also provide comprehensive empirical evidence showing that these residual networks are easier to optimize, and can gain accuracy from considerably increased depth. On the ImageNet dataset the residual nets were evaluated with a depth of up to 152 layers — 8× deeper than VGG nets but still having lower complexity.
```
Comment (simoncorstonoliver) [inline models/image_classification/resnet/README.md]: "Ease the training" -- do you mean reduce training time?

lower complexity??

File: models/image_classification/resnet/README.md [hunk 9]
```diff
@@ -36,66 +41,15 @@ All pre-trained models expect input images normalized in the same way, i.e. mini
...
-mxnet-model-server --models resnet18v1=https://s3.amazonaws.com/onnx-model-zoo/resnet/resnet18v1/resnet18v1.model
-```
-
-* **Run Prediction**:
-```bash
-curl -O https://s3.amazonaws.com/model-server/inputs/kitten.jpg
-curl -X POST http://127.0.0.1:8080/resnet18v1/predict -F "data=@kitten.jpeg"
-```
-For inference requests with all above ResNet models, Model Server expects the image to be passed in the data variable, which is the input layer's name in the model. In the previous example this was data=@kitten.jpeg.
+The post-processing involves calculating the softmax probablility scores for each classes and sorting them to report the most probable classes. Check [imagenet_postprocess.py](../imagenet_postprocess.py) for code.
```
Comment (simoncorstonoliver) [inline models/image_classification/resnet/README.md]: for each class

File: models/image_classification/squeezenet/README.md [hunk 10]
```diff
@@ -1,17 +1,18 @@
 # SqueezeNet
-SqueezeNet is a small CNN which achieves AlexNet level accuracy on ImageNet with 50x fewer parameters. SqueezeNet requires less communication across servers during distributed training, less bandwidth to export a new model from the cloud to an autonomous car and more feasible to deploy on FPGAs and other hardware with limited memory. The models perform image classification - they take images as input and classifies the major object in the image into a set of pre-defined classes. They are trained ImageNet dataset which contains images from 1000 classes. SqueezeNet models are highly efficient in terms of size and speed while providing good accuracies. This makes them ideal for platforms with strict constraints on size.
+
+## Use cases
+SqueezeNet models perform image classification - they take images as input and classifies the major object in the image into a set of pre-defined classes. They are trained ImageNet dataset which contains images from 1000 classes. SqueezeNet models are highly efficient in terms of size and speed while providing good accuracies. This makes them ideal for platforms with strict constraints on size.
```
Comment (simoncorstonoliver) [inline models/image_classification/squeezenet/README.md]: classifies --> classify

are trained on the ImageNet dataset

File: models/image_classification/mobilenet/README.md (hunk 6 above)
Comment (abhinavs95) [reply models/image_classification/mobilenet/README.md]: No input image falls in [0,255] and is normalized as x_norm = x/255 which is in [0,1].

File: models/image_classification/resnet/README.md (hunk 8 above)
Comment (abhinavs95) [reply models/image_classification/resnet/README.md]: No it means ease of convergence during training.. its because ResNet uses residual blocks which contain identity mappings from shallower layers.. this ensures that the training error is not more than the shallower network.. prevents exploding of training error for deeper nets

**Comment by ankkhedia** [comment]
Looks good to me now. Approving the request. Seems like all the issues got answered.

**Comment by ankkhedia** [comment]
@simoncorstonoliver - seems like all the concerns got addressed. Please feel free to raise an issue if something got missed.
